src/app/components/wrapper-components/online-game-wrapper/online-game-wrapper.quarto.component.spec.ts

```diff
@@ -1562,11 +1563,11 @@ describe('OnlineGameWrapperComponent of Quarto:', () => {
                     player: 0,
                 },
                 typeGame: 'P4',
-                playerZero: 'who is it from who cares',
+                playerZero: UserMocks.CREATOR_MINIMAL_USER,
                 turn: 1,
                 listMoves: [1],
                 result: MGPResult.UNACHIEVED.value,
-                playerOne: 'Sir Meryn Trant',
```

Comment:
hé non, les références à Game of Throne :|

Reply:
tu peux changer dans les UserMocks si tu veux les préserver :D